Components/SinglePokemon.tsx

```diff
@@ -83,7 +80,7 @@ export const SinglePokemon = ({ name }: Props) => {
           <IconButton
             icon={isFavourite ? blackHeart : whiteHeart}
             size={20}
-            onPress={() => handleFavourite(!isFavourite)}
+            onPress={() => handleFavourite()}
```

Comment:
Nie ma potrzeby tworzenia nowej funkcji tutaj
```suggestion
            onPress={handleFavourite}
```